raw-mm.dict.lua

```diff
@@ -278,7 +278,7 @@ codepaste.can_refill = function()
   return not ((affs.hemiplegyright and affs.hemiplegyleft) or affs.paralysis)
 end
 
-#for _, item in ipairs{"faeleaf", "myrtle", "coltsfoot", "steam"} do
+#for _, item in ipairs{"steam"} do
```

Comment:
I'd remove this `#for` and replace $(item) with `steam` that was within the old loop. It would make code easier to understand for someone new who comes and reads this code.